addons/hr_recruitment_survey/security/hr_recruitment_survey_security.xml

```diff
@@ -5,9 +5,19 @@
             <field name="implied_ids" eval="[Command.link(ref('survey.group_survey_user'))]"/>
```

Comment:
That's strange that recruitment interviewers gain survey user rights. Not sure they always do surveys (seems more like a manager thing to me, interviewers just use documents). Moreover it gives them access to all non appraisal surveys, which is strange to me.

To me it is more a customization than a real requirement: you can do it easily in your db if you want, but I would not consider it standard.

cc @sofiagvaladze

Reply:
And/Or give them read access to surveys they are set as interviewers, to keep things simple (linked to survey_user_input_rule_recruitment_interviewer rule: we could give them specific rules instead of survey user group).

Reply:
My reasoning back than was the following. On the applicant, we have "send interview" button which redirects to `model_survey_invite` and that model is only accessible for `group_survey_user`.  That's why interviewer includes survey_user.  And we want interviewer to be able to send surveys.

In appraisal it works because we have 'appraisal.ask.feedback'. And maybe doing something like this  for recruitment interviewer, to share surveys, is a bit of an overkill. At least that was my thinking 

Maybe there is an other way and I am missing it. :thinking: 

cc: @tde-banana-odoo

Reply:
I tried do that way and I recall what was the issue back then 

So, I granted to interviewer all the rights that I need for sending a survey (including on survey.invite). And when I do it, I get this access error

```
Document type: Survey (survey.survey)
Operation: read
User: 6
Fields:
- user_input_ids (allowed for groups 'Surveys / User')
```
This is because of this

https://github.com/odoo/odoo/blob/d2db535ec10e24713fcc2329927ce080534346ce/addons/survey/wizard/survey_invite.py#L72-L75

The user_input_ids has group on it
https://github.com/odoo/odoo/blob/21c25a7ccd0ba2d6574ddbcfbcf50dbbc03a1e6c/addons/survey/models/survey_survey.py#L102

Would it be possible to put sudo in 
https://github.com/odoo/odoo/blob/21c25a7ccd0ba2d6574ddbcfbcf50dbbc03a1e6c/addons/survey/wizard/survey_invite.py#L75
like this -> 

wizard.existing_partner_ids = list(set(wizard.survey_id.**sudo**()user_input_ids.partner_id.ids) & set(wizard.partner_ids.ids))

if so, we should be able to decouple recruitment.interviewer rights fully from survey rights without much hustle.. especially if we could use survey.invite :crossed_fingers: 

And if that sounds good do you and if you don't mind putting sudo in _compute_existing_partner_ids as suggested above, I can provide the code in hr_recruitment_survey side real quick 

What do you folks think?

cc: @tde-banana-odoo @lm4649

Reply:
Seems doable. We just have to ensure the rights on survey.invite are correctly set, to avoid letting people read "existing_partner_ids" on all possible invites (should probably have a rule on survey_id, at least for "not survey people" like recruitment guys, survey_id.applicant_id != False).

Reply:
@lm4649 I will make the changes in `hr_recruitment_survey` and force push here, if you don't mind? :smile: And if you approve, you can merge it, leave separate, rewrite or do whatever with it

Or otherwise I can also just make separate branch based on your branch and push there and you can cherry pick the commit.
I will do it by the end of the day :rocket: